src/components/ContactCard/ContactForm/fieldsConfig.jsx

```diff
@@ -0,0 +1,64 @@
+export const fields = [
+  {
+    name: 'givenName',
+    icon: 'people',
+    type: 'text'
+  },
+  {
+    name: 'familyName',
+    icon: null,
+    type: 'text'
+  },
+  {
+    name: 'company',
+    icon: 'company',
+    type: 'text'
+  },
+  {
+    name: 'jobTitle',
+    icon: null,
```

Comment:
https://github.com/cozy/cozy-contacts/pull/652/files#diff-93d1780bbd68ccd2a233f32e21567e3dR78 it seems that we have a `people` icon here, no?

Reply:
non, pas d'icone pendant l'utilisation du formulaire (création et édition d'un contact) par contre il faut une icone quand on consulte la fiche. C'est normal.